hr_expense_hide_taxes/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Hr Expense Hide Taxes",
+    "summary": """
+        Hide the 'tax_ids' fields
```

Comment:
Set a more human-like text like
```suggestion
        Hide taxes field for expenses
```